internal/native/display.go

```diff
@@ -99,18 +99,21 @@ func (n *Native) DisplaySetRotation(rotation uint16) (bool, error) {
 
 // UpdateLabelIfChanged updates the label if the text has changed
 func (n *Native) UpdateLabelIfChanged(objName string, newText string) {
-	l := n.lD.Trace().Str("obj", objName).Str("text", newText)
+	logger := GetDisplayLogger().
```

Comment:
for short durations (e.g. inside a method) it's okay to use the logger repeatedly... but we need to be vary careful for callbacks, goroutines, selects, and for {} loops... because they need to ask for their own logger at time of _use_